easybuild/easyconfigs/h/HDF5/HDF5-1.10.1-intel-2018.01.eb

```diff
@@ -0,0 +1,23 @@
+name = 'HDF5'
+version = '1.10.1'
+
+homepage = 'https://support.hdfgroup.org/HDF5/'
+description = """HDF5 is a data model, library, and file format for storing and managing data.
+ It supports an unlimited variety of datatypes, and is designed for flexible
+ and efficient I/O and for high volume and complex data."""
+
+toolchain = {'name': 'intel', 'version': '2018.01'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-%(version_major_minor)s/hdf5-%(version)s/src']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['048a9d149fb99aaa1680a712963f5a78e9c43b588d0e79d55e06760ec377c172']
+
+patches = ['HDF5-%(version)s-mpiifort-2018.1.163-fix.patch']
```

Comment:
@reedts This patch is not tied to a specific `impi` version, only to HDF5 as far as I can see.

So, please reuse the existing patch `HDF5-1.10.1-mpiifort-2018.0.128-fix.patch` (which should probably be named slightly differently, but fine, leave it as is).

There's no need to copy the existing patch only to change the filename.

Also, include a checksum for the patch too (use `eb --inject-checksums`).